integr8ly/keycloak-operator
# fix: Fix issue with double keycloak CR update in reconcile

## Motivation
Unable to run keycloak operator locally. The keycloak resource was being updated twice in the one reconcile loop.

## What
Removing one of the places the keycloak CR was being updated. Also, tidied up the various copies of the keycloak CR in the reconcile loop.

## Verification Steps

1. deploy keycloak-operator using this branch or the image `quay.io/integreatly/keycloak-operator:f11ede5`
2. create a keycloak realm using the KeycloakRealm_min template
3. create a keycloak CR using the keycloak_min template
4. see keycloak deploy successfully

## Checklist:


## Progress
